src/AvRadio.js

```diff
@@ -0,0 +1,57 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import {Input, FormGroup, Label} from 'reactstrap';
+import AvInput from './AvInput'
+
+let radioPropTypes = Object.assign({}, AvInput.propTypes);
+delete radioPropTypes.name;
+
+export default class AvRadio extends AvInput {
+
+  static contextTypes = Object.assign({}, AvInput.contextTypes, {
+    Group: PropTypes.object.isRequired,
+  });
+
+  static propTypes = radioPropTypes;
+
+  constructor (props) {
+    super(props);
+
+    this.onChangeHandler = ::this.onChangeHandler;
+  }
+
+  onChangeHandler () {
+    this.context.Group.update(this.props.value);
```

Comment:
You should also check if there is a `onChange` prop and give that a call if there is.